packages/api/src/command/medical/patient/converage-assessment-create.ts

```diff
@@ -22,38 +23,89 @@ export async function createCoverageAssessments({
   facilityId: string;
   patientCreates: PatientCreateCmd[];
 }): Promise<void> {
-  const chunkSize = 50;
+  const { log } = out(`createCoverageAssessments - cxId ${cxId}`);
   const patients: Patient[] = [];
 
-  const pdChunks = chunk(patientCreates, chunkSize);
-  for (const chunk of pdChunks) {
-    await sleep(delayTime.asMilliseconds());
-    const createPatients: Promise<Patient>[] = [];
-    for (const patient of chunk) {
-      createPatients.push(createOrUpdatePatient(patient));
+  async function createOrUpdatePatientWrapper({
+    patientCreateCmd,
+    patients,
+    log,
+  }: {
+    patientCreateCmd: PatientCreateCmd;
+    patients: Patient[];
+    log: typeof console.log;
+  }): Promise<void> {
+    try {
+      const patient = await createOrUpdatePatient(patientCreateCmd);
+      patients.push(patient);
+    } catch (error) {
+      const msg = `Failed to create or update patient. Cause: ${errorToString(error)}`;
+      log(msg);
+      capture.error(msg, {
+        extra: {
+          cxId,
+          facilityId,
+          context: "coverage-assessment.create",
+        },
+      });
```

Comment:
ditto: have this outside the wrapper, so we run it once per call of `createCoverageAssessments`

Reply:
Done.